Origin: commit:aba7c7b89fcbbfa516d86f50f7b89a593ca95303
Author: Ray Strode <rstrode@redhat.com>
Subject: terminal: Don't set ISTRIP in terminal attributes

In certain error paths plymouth would erroneously set
ISTRIP on the terminal.  This is bogus in the same way
the changes fixed by commit ea394383c54dda771dcf59e7ec11280fa945280b
were bogus.

Index: b/src/libply-splash-core/ply-terminal.c
===================================================================
--- a/src/libply-splash-core/ply-terminal.c
+++ b/src/libply-splash-core/ply-terminal.c
@@ -218,7 +218,7 @@
    */
   if (!terminal->original_term_attributes_saved || !(terminal->original_term_attributes.c_lflag & ICANON))
     {
-      term_attributes.c_iflag |= BRKINT | IGNPAR | ISTRIP | ICRNL | IXON;
+      term_attributes.c_iflag |= BRKINT | IGNPAR | ICRNL | IXON;
       term_attributes.c_oflag |= OPOST;
       term_attributes.c_lflag |= ECHO | ICANON | ISIG | IEXTEN;
 
Index: b/src/main.c
===================================================================
--- a/src/main.c
+++ b/src/main.c
@@ -1766,7 +1766,7 @@
 
     tcgetattr (fd, &term_attributes);
 
-    term_attributes.c_iflag |= BRKINT | IGNPAR | ISTRIP | ICRNL | IXON;
+    term_attributes.c_iflag |= BRKINT | IGNPAR | ICRNL | IXON;
     term_attributes.c_oflag |= OPOST;
     term_attributes.c_lflag |= ECHO | ICANON | ISIG | IEXTEN;
 
